src/foam/nanos/crunch/MinMaxCapability.js

```diff
@@ -71,6 +71,7 @@ foam.CLASS({
 
           UserCapabilityJunction ucJunction =
             crunchService.getJunction(x, ccJunction.getTargetId());
+          if ( ucJunction.getStatus() == AVAILABLE ) continue;
```

Comment:
think we can and should delete the line below that does the null check

Reply:
Agreed - thanks!